lib/modules/platform/github/readme.md

```diff
@@ -96,7 +96,8 @@ When Renovate runs against repositories on `github.com`, and the environment var
 - `rubygems.pkg.github.com`
 
 <!-- prettier-ignore -->
-!!! warning Users have reported that this feature is not working correctly, so it has been reverted to experimental mode.
+!!! warning
```

Comment:
I should probably resolve this comment, with this PR:

- https://github.com/renovatebot/renovate/discussions/26475#discussioncomment-8141037

But I don't know where/what to change in this document, just from reading the linked discussion. Can a maintainer explain what's needed?

Reply:
We should say that it does not appear to work with app tokens so has been put behind this feature flag